src/index.html

```diff
@@ -1,13 +1,319 @@
+      <section class="categories">
+        <div class="container">
+          <div class="categories__content">
+            <h2 class="categories__title">Sound. Vision. It's your pick.</h2>
+
+            <a href="#" class="categories__link">
+              <img
+                src="./img/portable_speaker_1.png"
+                alt="portable speaker"
+                class="categories__img"
+              >
+            </a>
+
+            <a href="#" class="categories__link">
+              <img
+                src="./img/portable_speaker_2.png"
+                alt="portable speaker"
+                class="categories__img categoies__img--1"
+              >
+            </a>
+
+            <h3 class="categories__header">Portable speakers</h3>
+
+            <a href="#" class="categories__link">
+              <img
+                src="./img/earphones_1.png"
+                alt="earphones"
+                class="categories__img"
+              >
+            </a>
+
+            <a href="#" class="categories__link">
+              <img
+                src="./img/earphones_2.png"
+                alt="earphones"
+                class="categories__img categoies__img--2"
+              >
+            </a>
+
+            <h3 class="categories__header">Earphones</h3>
+
+            <a href="#" class="categories__link">
+              <img
+                src="./img/smart_home_1.png"
+                alt="smart home"
+                class="categories__img"
+              >
+            </a>
+
+            <a href="#" class="categories__link">
+              <img
+                src="./img/smart_home_2.png"
+                alt="smart home"
+                class="categories__img"
+              >
+            </a>
+
+            <h3 class="categories__header">Smart Home</h3>
+
+            <a href="#home" class="categories__link-button">
+              <button class="button">View all</button>
+            </a>
+          </div>
+        </div>
+      </section>
+      <!-- #endregion -->
+      <!-- #region SPECIAL OFFER-->
+      <section class="special-offer">
+        <div class="special-offer__background"></div>
+        <div class="container">
+          <div class="special-offer__content">
+            <h2 class="special-offer__title">Timeless, for 50 years and counting</h2>
+
+            <p class="special-offer__text">Bang & Olufsen is expanding its Recreated Classics Program with the launch of
+              Beosystems - a limited edition music system that transcends time by bridging
+              the gap between one of our iconic designs from 1972 and today’s cutting-edge
+              digital technology.
+            </p>
+          </div>
+        </div>
+      </section>
+      <!-- #endregion -->
+      <!-- #region CONTACTS-->
+      <section class="contacts">
+        <div class="container">
+          <div class="contacts__content">
+            <h2 class="contacts__title">Contact us</h2>
+
+            <form
```

Comment:
fields should be required, I shouldn't be able to send empty form

Reply:
Done.